pyspark/executor/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# the result of this command is msft/spark-py:build
+wget https://downloads.apache.org/spark/spark-3.1.2/spark-3.1.2-bin-hadoop3.2.tgz && \
+tar -xf spark-3.1.2-bin-hadoop3.2.tgz && \
+./spark-3.1.2-bin-hadoop3.2/bin/docker-image-tool.sh -r msft -t build -p ./Dockerfile build
```

Comment:
This brings me the question of what is the proper location of this image? Should it live in its own repo? cc @TomAugspurger